entrypoint/upgradecluster/upgradecluster_suite_test.go

```diff
@@ -10,18 +10,20 @@ import (
 
 	"github.com/rancher/distros-test-framework/config"
 	"github.com/rancher/distros-test-framework/pkg/customflag"
+	"github.com/rancher/distros-test-framework/pkg/qase"
 	"github.com/rancher/distros-test-framework/shared"
 )
 
 var (
+	qaseReport = os.Getenv("REPORT_TO_QASE")
 	kubeconfig string
 	flags      *customflag.FlagConfig
 	cluster    *shared.Cluster
 )
 
 func TestMain(m *testing.M) {
 	flags = &customflag.ServiceFlag
-	flag.Var(&flags.InstallMode, "installVersionOrCommit", "Upgrade with version or commit")
+	flag.Var(&flags.UpgradeMode, "installVersionOrCommit", "Upgrade with version or commit")
```

Comment:
https://github.com/rancher/distros-test-framework/pull/177/files#r1825860085

Reply:
Also I think we need to rename `installVersionOrCommit`.. it can be a separate PR